tests/patmat/t9232.check

```diff
@@ -1,5 +1,5 @@
 ./tests/patmat/t9232.scala:13: warning: match may not be exhaustive.
-It would fail on the following input: Node1(_), Node2()
+It would fail on the following input: Node2()
```

Comment:
OK, in cases where you don't know statically what an extractor match I think it's best to act as if it matched nothing